src/server/services/MediaResourceServices.js

```diff
@@ -56,13 +56,19 @@ class MediaResourceServices {
 
     switch (parseInt(mediaType)) {
       case constants.MEDIA_TYPE.BRUINCAST:
-        mediaCollectionName = 'bruincastmedia';
+        mediaCollectionName = constants.collectionMap.get(
```

Comment:
use something like const { MEDIA_TYPE, COLLECTION_TYPE, collectionMap } = require('constants'); so that you don't need to say constants.something for everything.